pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_creating-a-product.adoc

```diff
@@ -12,23 +14,23 @@ A product must exist in {ProductName} before you can assign a product name to th
 .Procedure
 
 . Navigate to menu:Products[New Product].
-
-. Enter the following information:
-  Product Name:: The text that appears in {ProductShortName}, the document preview, and in user-facing content.
-  Product UrlFragment:: This string appears in the user-facing URL for any content that belongs to this product and version. A URL fragment can include letters, numbers, hyphens, periods and underscores.
-  Product Description (optional):: A description of the product.
-
+. Enter the following information, items marked with a star are mandatory:
+  Product Name *:: The text that appears in {ProductShortName}, the document preview, and in user-facing content. This name will be substituted wherever the `:pantheonproduct:` attribute is used in user-facing content that is generated by {ProductShortName}. 
+  Product URL Fragment *:: This string appears in the user-facing URL for any content that belongs to this product and version. A URL fragment can include letters, numbers, hyphens, periods and underscores.
+  Product Description:: A description of the product.
+  Product Version *:: The product version. This value will be substituted wherever the `:pantheonversion:` attribute is used in user-facing content that is generated by {ProductShortName}.
+  Version URL Fragment *:: This string appears in the user-facing URL for any content that belongs to this product and version. A URL fragment can include letters, numbers, hyphens, periods and underscores.
 . Click btn:[Save].
 
-After creating a product, you need to create a product version.
-
 .Example
 
-Product Name:: .NET Core
-Product UrlFragment:: `net_core`
-Product Description:: Documentation for the .NET Core framework.
+* Product Name: .NET Core
```

Comment:
Use parallelism wherever possible. The definition list above makes sense here too.